src/content/docs/installation/method/nix.mdx

```diff
@@ -0,0 +1,23 @@
+---
+title: Nix Installation
+---
+import Aside from '@components/Aside.astro'
+export const partial = true;
+
+Daytona is available on nixpkgs unstable as [`daytona-bin`](https://search.nixos.org/packages?channel=unstable&show=daytona-bin).
+
+<Aside type="caution">
+The version of Daytona available in nixpkgs may differ from the version documented here.
```

Comment:
@osslate yeah, remove that note please.

Even if the nixpkgs version is not synced up with the GH release, the note doesn't matter since the docs are versioned as well.

Reply:
@Tpuljak some context around why I added this:

Since the docs match specific Daytona versions, for accuracy we should be instructing the user to install the corresponding version of Daytona. We can do this with the installation script using `DAYTONA_SERVER_VERSION`, but there's no way to specify the exact version of a Daytona installation on Nix/Homebrew currently. The note is intended as a flag to say "be aware that the docs you're looking at may not be accurate for the version of Daytona available today using this installation method".

For example, today, a user might go through the Nix instructions, try to configure a container registry, and discover that the commands don't work because the Nix version is a few versions behind. If users are reading the `latest` docs without this note, they might (incorrectly) assume that the version they install using Nix matches the docs they're reading, and be confused when things don't work.

I can rewrite it to clarify what I meant, but I'd argue against removing a note here entirely. It's useful context that will likely prevent headaches for new users.

As for versioning, right now users have no way of selecting which version of the docs they're looking at. @gdraganic and I need to work on a version picker and deployment solution. Until that's sorted, people won't be able to navigate to the docs for the version they might install from Nix.

Reply:
@osslate after installing Nix/Homebrew you still can hit` daytona version` command, no?

If so the note is not needed

Reply:
We can tackle the version selector in another issue, but I still agree with Ivan.

Users should be aware which version of Daytona they're running and they can select the appropriate version of the docs.

Reply:
Great. lets remove this note for npw and @osslate you can open an issue about create a version selector in docs.

Reply:
Opened #68 to track the version selector

Reply:
Resolved in [c7ba825](https://github.com/daytonaio/docs/pull/59/commits/c7ba825f29c3612fa5e22abc99280521f5141dae)